locales/base/translation.json

```diff
@@ -591,7 +591,8 @@
   "sendAndInvite": "Send & Invite",
   "inviteAndSendDialog": {
     "title": "Finish inviting {{name}} to Valora",
-    "body": "Once they join & connect a phone number, you both receive {{amount}} {{currency}}. Click below to send and customize your invite.",
+    "bodyWithRewards": "Once they join & connect a phone number, you both receive {{amount}} {{currency}}. Click below to send and customize your invite.",
+    "body": "Share an invitation message with {{name}} to join {{appName}}. Once they join & connect a phone number, they'll receive your payment.",
```

Comment:
this "body" key is as it was prior to https://github.com/valora-inc/wallet/pull/2492 to avoid a breaking change. i personally think this copy is still fine for users who are not part of the invite rewards beta and we can always update via OTA (feels out of scope to request changes for this last minute tbh)